src/schema-to-typebox.ts

```diff
@@ -32,8 +33,23 @@ import {
 
 type Code = string;
 
+export type SupportedFiletypes = "CJS" | "TS" | "ESM";
```

Comment:
Can we adapt this to something like this?

```suggestion
export type OutputType = "js-cjs" | "js-esm" | "ts-esm";
```